src/Resources/skeleton/validator/Validator.tpl.php

```diff
@@ -9,7 +9,7 @@ class <?= $class_name ?> extends ConstraintValidator
 {
     public function validate($value, Constraint $constraint)
     {
-        /* @var $constraint \<?= $constraint_class_name ?> */
+        /** @var <?= $constraint_class_name ?> $constraint */
```

Comment:
```suggestion
        /* @var <?= $constraint_class_name ?> $constraint */
```

Reply:
But that would make my fix completely pointless. If I'm not using docblock phpstorm or phpstan won't recognise this as a typehint, just as a regualr comment...
![image](https://user-images.githubusercontent.com/1424582/111902899-d7060200-8a3f-11eb-9d1c-dc998a43e370.png)
vs
![image](https://user-images.githubusercontent.com/1424582/111902914-e9803b80-8a3f-11eb-999d-895465bbc882.png)

Reply:
@jrushlow could you help me with this one pls?